nipype/interfaces/fsl/model.py

```diff
@@ -1134,28 +1134,28 @@ class L2Model(BaseInterface):
 
     def _run_interface(self, runtime):
         cwd = os.getcwd()
-        mat_txt = ['/NumWaves       1',
-                   '/NumPoints      %d' % self.inputs.num_copes,
-                   '/PPheights      %e' % 1,
+        mat_txt = ['/NumWaves   1',
+                   '/NumPoints  {}'.format(self.inputs.num_copes),
+                   '/PPheights  1',
```

Comment:
Arguments:
* It uses the appropriate types for the respective  parameters the previousversion does not
* It is easier to read at both the nipype code level as well as at the level of the files it produces - the previous version is considerably more confusing for the casual observer
* It resembles the examples given by FSL - the previous version does not
* It only uses string formatting when it is needed - the previous version does not

Evidence for it working:
* I am using the newer version, and since it's a one-purpose function it is unlikely there might be any context in which it would fail if it did not already do so.

I would go even further and say that since we are not using the FSL backend in the way FSL expects us to - the old version would be more prone to break in the future.